.azure-devops/nightly.yml

```diff
@@ -35,10 +35,6 @@ parameters:
   type: object
   default: >
     {
-      Python37:
```

Comment:
This needs to be modified to the next minimum i.e. 3.8 so our nightly tests the min and max supported runtimes.

Reply:
I think I got everything now